libs/ledger-live-common/src/apps/listApps/v1.ts

```diff
@@ -252,6 +256,11 @@ const listApps = (transport: Transport, deviceInfo: DeviceInfo): Observable<List
         }
       }
 
+      const languageId: number = deviceInfo.languageId || 0;
```

Comment:
shouldn't we use `languageIds.english` instead of `0` here? It's the same value but 0 looks more arbitrary

Reply:
Done 👍